eng/versioning/version_client.txt

```diff
@@ -107,6 +107,7 @@ com.azure.cosmos.spark:azure-cosmos-spark_3-4_2-12;4.23.0;4.24.0-beta.1
 com.azure:azure-cosmos-encryption;2.5.0;2.6.0-beta.1
 com.azure:azure-cosmos-test;1.0.0-beta.5;1.0.0-beta.6
 com.azure:azure-cosmos-tests;1.0.0-beta.1;1.0.0-beta.1
+com.azure.cosmos.kafka:azure-cosmos-kafka-connect;2.0.0-beta.1;2.0.0-beta.1
```

Comment:
Since we are changing the `artifactId` (from previous version), we might have to start with v1.0.0-beta.1 semantically.

Reply:
Fixed